### PR TITLE
Cabal 3.2 support

### PR DESCRIPTION
**Pull request by larskuhtz**

### CODE REVIEW

**Comment by fosskers** [comment]
Let's switch to Github CI.

**Comment by larskuhtz** [comment]
It may also be time again to drop support for some older cabal versions.

**Comment by fosskers** [comment]
And `base` versions? Good thing I just wrote [a handy post](https://www.fosskers.ca/en/blog/base).

File: configuration-tools.cabal [hunk 1]
```diff
@@ -95,7 +95,7 @@ library
       , ansi-wl-pprint >= 0.6
       , attoparsec >= 0.11.3.4
       , base >= 4.8 && < 5
-      , base-unicode-symbols >= 0.2.2.4
+      , base-unicode-symbols >= 0.2.2.4 < 0.2.4
```
Comment (larskuhtz) [inline configuration-tools.cabal:98]:
```suggestion
      , base-unicode-symbols >= 0.2.2.4 && < 0.2.4
```

**Pull request merged**